integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/MicroRegistryManager.java

```diff
@@ -119,6 +120,24 @@ public void updateResource(String resourcePath, String sourcePath, boolean backu
         }
     }
 
+    /**
+     * Function is to check if the registry resources are exist in filepath
+     *
+     * @param sourcePath path to registry resource
+     * @param resourcePath path to resource file path from registry resource
+     * @param filename registry filename
+     */
+
+    public boolean checkResourceExist(String sourcePath, String resourcePath, String filename) throws Exception {
+        try {
+            String path = sourcePath + resourcePath + filename;
+            return path.contains(filename);
```

Comment:
This does not evaluate whether the resource exists in the file path. Here you are concatenating a string like a + b + c and then you are checking whether the c contains in the concatenated string.